packages/php-wasm/compile/php/Dockerfile

```diff
@@ -2126,6 +2157,7 @@ RUN set -euxo pipefail; \
 	-s MIN_NODE_VERSION=200900 \
 	-s INITIAL_MEMORY=1024MB \
 	-s ALLOW_MEMORY_GROWTH=1 \
+	-s STACK_SIZE=1MB \
```

Comment:
I have conducted tests for 16MB, 8MB, 4MB, 2MB, and 1MB. I can potentially lower it again, but it's better to force it, just [for a few KB](https://github.com/emscripten-core/emscripten/issues/18284#issuecomment-1356376398).

Do you think we should pass this as a Docker argument?